tests/common/nebula_test_suite.py

```diff
@@ -146,10 +146,19 @@ def use_student_space(self):
     def create_nebula_clients(self):
         self.client_pool = ConnectionPool(ip=self.host,
                                           port=self.port,
+                                          socket_num=16,
                                           network_timeout=0)
         self.client = GraphClient(self.client_pool)
         self.client.authenticate(self.user, self.password)
 
+    @classmethod
+    def spawn_nebula_client(self):
+        return GraphClient(self.client_pool)
```

Comment:
I don't know why you need to define `spawn_nebula_client` and `close_nebula_client` again, I think you can update `self.user` and `self.password` in subclass and reuse  `create_nebula_clients` function.

Reply:
The main purpose is to avoid modify the origin configuration in testing.

Reply:
And `create_nebula_clients` will recreate a connection pool.